packages/studio-be/src/studio/nlu/nlu-service.ts

```diff
@@ -163,6 +163,29 @@ export class NLUService {
     void bot.train(language)
   }
 
+  public async downloadAndSaveModelWeights(botId: string) {
+    const bot = this._bots[botId]
+    if (!bot) {
+      throw new BotNotMountedError(botId)
+    }
+
+    const botConfig = await this.configProvider.getBotConfig(botId)
+
+    if (!botConfig.nluModels) {
+      throw new Error('Missing NLU models. Bot is not trained.')
+    }
+
+    const modelsFolder = 'models'
+    await this.ghost.forBot(botId).deleteFolder(modelsFolder)
```

Comment:
We delete the models folder before saving them, in case there was some already.

Reply:
:+1: